### PR TITLE
updated nuxt & vuetify to latest and fixed eslint error (no more eslint error)

### PR DESCRIPTION
**Pull request by Codefa**
Updated all dependencies & devDependencies to latest 
I Personally tested it without a-la-carte and it works well, also i tested with a-la-carte and it works well but i did some changes in the code, here i will show you two demos check it it works fine with new updated code with nuxt generate / nuxt build, for a-la-carte i need to some tests more and i need some helps too, 
for a-la-carte i used vuetify-loader and followed its docs works great without any eslint errors
 ### **demo sites** 
**Normal Vuetify Nuxt template ( without a-la-carte )**
[](url)https://nuxt-test-vuetify.herokuapp.com/

**with a-la-carte**
[](url)https://nuxt-vuetify-a-la-carte-test.herokuapp.com/

to test my repository by adding this command 
`$ vue init Codefa/nuxt my-project`

### CODE REVIEW

File: template/package.json [hunk 1]
```diff
@@ -11,23 +11,23 @@
     "generate": "nuxt generate"
   },
   "dependencies": {
-    "nuxt": "^2.1.0",
-    "vuetify": "^1.2.6"
+    "nuxt": "^2.2.0",
+    "vuetify": "^1.3.2"
   },
   "devDependencies": {
     "babel-eslint": "^7.2.3",
```
Comment (XanderLuciano) [inline template/package.json]:
```suggestion
     "babel-eslint": "^10.0.1",
```

Update to `"babel-eslint": "^10.0.1",`

File: template/nuxt.config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export default {
+module.exports = {
```
Comment (XanderLuciano) [inline template/nuxt.config.js]:
```suggestion
exports default {
```

Why revert this line to ES5?

File: template/package.json [hunk 3]
```diff
@@ -11,23 +11,23 @@
...
-    "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-standard": "^3.0.1",
+    "eslint": "^5.7.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-loader": "^2.1.1",
+    "eslint-plugin-html": "^4.0.6",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^7.0.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-standard": "^4.0.0",
```
Comment (XanderLuciano) [inline template/package.json]:
```suggestion
    "eslint-plugin-standard": "^4.0.0",
    "eslint-plugin-vue": "^4.0.0",
```

Maybe consider adding a Vue linter? 
https://github.com/vuejs/eslint-plugin-vue

**Comment by XanderLuciano** [comment]
Your PR covers my previous PR + more so I closed it in favor of this.

File: template/nuxt.config.js (hunk 2 above)
Comment (Codefa) [reply template/nuxt.config.js]: we can use ES6 but still nuxt official uses ES5 that's why just sticking to the official one don't want to face more bugs you know

File: template/package.json (hunk 3 above)
Comment (Codefa) [reply template/package.json]: we are using standard eslint javacript rules eslint-plugin-vue required when we use prettier and use plugin:vue/essential or recommend i suggest to follow the standard javscript eslint rules

File: template/package.json (hunk 1 above)
Comment (Codefa) [reply template/package.json]: @XanderLuciano Thank you updated

**Comment by Codefa** [comment]
> Your PR covers my previous PR + more so I closed it in favor of this.

@XanderLuciano can you test my repoitory please select A la carte and tet A la carte works or not i'm not sure about that please check it [](url)https://github.com/Codefa/nuxt
 `$vue init Codefa/nuxt appname`

**Pull request merged**